etc/kayobe/kolla/config/monasca/notification.conf

```diff
@@ -0,0 +1,8 @@
+[notification_types]
+enabled = slack
+
+[slack_notifier]
+message_template: "/etc/monasca/slack_template.j2"
+timeout: 10
+ca_certs: "/etc/ssl/certs/ca-bundle.crt"
+insecure: False
```

Comment:
should these be key = value INI format?

Reply:
Oops, for consistency yes, although it seems to work fine.